ports/raspberrypi/audio_dma.c

```diff
@@ -313,13 +369,22 @@ void audio_dma_stop(audio_dma_t *dma) {
         if (dma_channel_is_busy(channel)) {
             dma_channel_abort(channel);
         }
+
+        // Write a zero as the last sample. This stops any PWM output.
```

Comment:
Should this be the quiescent value and only zero after deinit? I don't think you want to change this here because you do set it to quiescent value later. It may cause a click then.

Reply:
I think I can take this away because I just changed the `PWMAudio` to set the duty cycle to zero on stop. That may still cause a click, but otherwise there is a quiet white noise on the speaker which is quite annoying.

Reply:
I deleted those lines. I thought originally the non-zero values were causing noise, but that was long ago in the debugging process.

I think deferring a ramp-down to avoid clicks if necessary to another PR is OK.